nixos/modules/programs/bat.nix

```
@@ -26,6 +26,14 @@ let
   settingsFormat = pkgs.formats.keyValue { listsAsDuplicateKeys = true; };
   inherit (settingsFormat) generate type;
 
+  recursiveToString = value:
+    if isList value then
+      map recursiveToString value
+    else if builtins.isBool value then
```

```suggestion
    else if isBool value then
```
Inherit `isBool` from `lib`. This is purely stylistic, as we avoid having to call for `builtins`.

---

sure

---

You...uh, also need to add it to the `inherit` statement at the top of the file.

---

done